File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.swift
import Foundation

let n = readLine()
let clothes = readLine()
let pen = readLine()

func solution() {
    guard let n = n, let clothes = clothes, let pen = pen else { return }
    let N = Int(n)!
    let applicantSize = clothes.split(separator: " ").compactMap{ Int($0) }
    let penList = pen.split(separator: " ").compactMap{ Int($0) }
    let (T, P) = (penList[0], penList[1])
    
    var count = 0
    
    for i in applicantSize {
        if i % T == 0 {
            count += i / T
        } else {
            count += i / T + 1
        }
    }
    
    print(count)
    print(N/P, N%P)
}

solution()

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
N = int(input())
applicant_size = list(map(int, input().split()))
T, P = map(int, input().split())

count = 0

for i in applicant_size:
    if i % T == 0:
        count += i//T
    else:
        count += (i//T + 1)

print(count)
print(N//P, N%P)